user/TASK/gimbal_task/gimbal_task.h

```diff
@@ -17,16 +17,16 @@
 
 
 /*************** Converts between motor position and degrees *****************/
-#define Motor_Ecd_to_Rad 0.000766990394f
+#define Motor_Ecd_to_Rad 0.000766990394f /* 2PI / 8191 */
 #define FALSE 0
 #define TRUE 1
 
 
 /****************************** PID Constants ********************************/
-#define pid_kp_yaw 40.0f
+#define pid_kp_yaw 250.0f
 #define pid_ki_yaw 0.0f
```

Comment:
Again, if 0 shouldn't it be an int?

Reply:
I don't think so.... I'm worried about casts occurring somewhere